src/ray/core_worker/core_worker.cc

```diff
@@ -366,22 +387,27 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
       /* retry_task_callback= */
       [this](TaskSpecification &spec, bool object_recovery, uint32_t delay_ms) {
         spec.GetMutableMessage().set_attempt_number(spec.AttemptNumber() + 1);
+        auto fn = [spec, this]() {
```

Comment:
Let's use a better name than fn (retry_task)?

Reply:
maybe it's better if we can make it a private method instead of lambda ?

Reply:
done